storage/redis_cluster_test.go

```diff
@@ -17,7 +17,7 @@ var rc RedisController
 func init() {
 	conf := config.Default
 
-	rc = RedisController{ctx: context.Background()}
+	rc = RedisController{ctx: context.Background(), reconnect: make(chan struct{}, 1)}
```

Comment:
Ah no, using NewController was fine, just keep rc as a pointer (e.g. just removing the copy).